blocks/library/cover-image/style.scss

```diff
@@ -7,7 +8,7 @@
 	display: flex;
 	justify-content: center;
 	align-items: center;
-
+	
```

Comment:
Not explicitly documented for CSS guidelines, but in general we should remove trailing whitespace.

PHP reference: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#remove-trailing-spaces

I also suggest downloading an EditorConfig plugin for your editor, which should simplify this task for you since we include a configuration file:

http://editorconfig.org/#download